resources/zib/terminology/CodeSystem-COD472VEKT-2.16.840.1.113883.2.4.3.11.22.472.xml

```diff
@@ -0,0 +1,26 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
```

Comment:
A new CodeSystem is added. This cannot do any harm I guess, but it is not inside the scope of ZIBFHIR-67. So double check if this is desired.

Reply:
I think this is due to https://nictiz.atlassian.net/browse/ZIB-2052, and I think this should be a separate ticket.

Reply:
Good, I see https://nictiz.atlassian.net/browse/ZIBFHIR-69 is made for this.

Reply:
Correct, the addition has been removed from this PR